src/govuk/components/accordion/accordion.mjs

```diff
@@ -404,21 +403,27 @@ Accordion.prototype.getButtonPunctuationEl = function () {
 export default Accordion
 
 /**
- * @typedef {object} AccordionTranslations
+ * Accordion translations
+ *
+ * @typedef {object} Translations
  *
  * Messages used by the component for the labels of its buttons. This includes
  * the visible text shown on screen, and text to help assistive technology users
  * for the buttons toggling each section.
- * @property {string} [hideAllSections] - The text content for the 'Hide all
+ * @property {Translation} [hideAllSections] - The text content for the 'Hide all
```

Comment:
Happy to leave it out?

Using a named `Translation` type would give us:

1. JSDoc global section **Translation**
2. Description about including string interpolation placeholders
3. Generation of JSDoc `@example` snippets into the documentation

Not done it yet, so it can wait

I felt like `{string}` lacked all that magic 🪄

Reply:
It's a bit light but I've added a description in the latest push

```mjs
/**
 * Translated message
 *
 * Supports ${} placeholders for string interpolation
 *
 * @typedef {string} Translation
 */
```

Reply:
<img width="408" alt="JSDoc Translation type preview showing new message" src="https://user-images.githubusercontent.com/415517/201046733-a37a7355-63ab-4391-ab6d-93c4cda8ed23.png">

Reply:
I'm going to remove **Translation** and **Translations**

Let's bring them back when we've got better JSDoc documentation with code `@examples` etc